extensions/src/ACRE2Core/stopZeusSpeaking.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "compat.h"
+#include "Types.h"
+#include "Macros.h"
+#include "Log.h"
+#include "IRpcFunction.h"
+
+#include "IServer.h"
+#include "Engine.h"
+
+#include "TextMessage.h"
+
+#include <string>
```

Comment:
I used `stopRadioSpeaking` and `stopIntercomSpeaking` as the base for this file, but as far as I can tell there is no reason for it to be in any of them.